src/functions-and-arrays.js

```diff
@@ -1,41 +1,90 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a >= b) {
+    return a;
+  }
+  return b;
```

Comment:
Little challenge for you here :)

Try to refactor this code (re-write it) using ternary operator.
In this case do not forget to return the result you get ;)
```
condition ? `will happen if conition is true ` : ` will happen if conition is false`
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator